components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -152,23 +149,106 @@ The default policies are: Viewers, Editors, Administrator, and Ingest.
+Without a policy statement allowing an operation for a resource assigned to a given project, a user cannot perform the actions of that policy.
+
+{{< info >}}
+Note that Chef Automate is currently limited to six projects while we continue to refine the user experience during this beta period.
+{{< /info >}}
+
+There are four steps to setting up projects:
+
+1. Define each project via the UI (or the command line if you prefer).
+   Start on the projects list page shown previously to create your projects.
+2. Create or modify IAM policy statements to restrict permissions to specific projects as needed.
+   *Every* statement must either indicate specific projects or specify the wildcard denoting all projects (`*`).
+   By default, any previously existing policies are automatically setup with that wildcard so they apply to all projects.
+3. Assign resources to projects (see [Assigning Resources to Projects]({{< relref "iam-v2-overview.md#assigning-resources-to-projects" >}})).
+4. Select the projects to filter in the UI.
+   Once you create some projects, use the **global project filter** in the top navigation bar to select one or more projects for viewing.
+   With no selection (the default), you will see resources for *all* projects.
+
+Note that, by default, Automate includes a *project-owner* role so the global admin may delegate much of these responsibilities for project management to others, to alleviate some of the burden for the global admin.
+
+### Assigning Resources to Projects
+
+There are two broad categories of resources that may be assigned to projects: your set of nodes feeding data into Chef Automate and the set of IAM resources themselves.
+Any of these that you do not explicitly assign to a project (as described next) are considered *unassigned* with respect to projects.
+At the start, none of your resources are assigned to any projects.
+These unassigned resources will be displayed either if you have made no selection in the global project filter, or if you have selected the `(unassigned)` designator.
+
+Assignable IAM resources include teams, tokens, policies, and roles.
+Teams and tokens may be assigned to projects directly in the UI; policies and roles still can only be assigned on the command-line (since you cannot create or modify those at all in the UI yet).
+
+Assigning your set of nodes to projects is done via the rules you create for the project, mentioned just briefly above.
+A rule specifies one or more conditions, and each condition specifies a set of attribute values that a node must satisfy to be assigned to the given project.
+Once you define your set of projects with their contained rules and conditions, you then use the **Update Projects** button on the main projects list page to apply those definitions to all your nodes.
+
+### Properties of a Project
+
+To assign a project to a set of nodes as mentioned above, the project needs a list of **rules**, where each rule describes a group of node characteristics.
```

Comment:
let's add somewhere in this section that a project is not required to have rules if it's only for managing resources like teams or tokens.